### ISSUE TITLE
API_REPORTER does not work on v8 Go flavor

### ISSUE THREAD

**Issue by dochang** [oxsecurity/megalinter#4219]
**Describe the bug**
API_REPORTER does not work on v8 Go flavor, but it does work on the main MegaLinter image.

**To Reproduce**
In this commit [81d03ce], I enable API_REPORTER and set `API_REPORTER_DEBUG` to true, on the Go flavor. There are no API_REPORTER related logs in the job [32444677906].

In the next commit [49eb482], I switch the docker image from the Go flavor to the main image. Now API_REPORTER works, you can see the logs in the job [32444740669].

[81d03ce]: https://github.com/dochang/koanfenv/commit/81d03ce4d8c4702f03add6fee6ace8aecb0acbe1
[32444677906]: https://github.com/dochang/koanfenv/actions/runs/11652901483/job/32444677906
[49eb482]: https://github.com/dochang/koanfenv/commit/49eb482fdd135234628809e0a7d2cbeccfbadb2d
[32444740669]: https://github.com/dochang/koanfenv/actions/runs/11652930149/job/32444740669

**Expected behavior**
API_REPORTER works on all flavors.

**Comment by nvuillam** [comment]
@dochang the issue has been solved in v8.1.0

If you are using the GitHub Action, there is still an issue with "v8" tag, please can you directly use v8.1.0 ?

**Comment by dochang** [comment]
Hi @nvuillam , now the Go flavor v8.1.0 WORKS in the job [32450221588].

[32450221588]: https://github.com/dochang/koanfenv/actions/runs/11655471993/job/32450221588

And I re-check the logs in the previous "v8" job, the docker image version is "v8.0.0".

```
 - Image Creation Date: 2024-08-19T21:32:01Z
 - Image Revision: c217fe8f7bc9207062a084e989bd97efd56e7b9a
 - Image Version: v8.0.0
```

**Comment by nvuillam** [comment]
@dochang thanks for the feedback with 8.1.0 :)
About v8 not targeting latest v8.x version, the problem is identified and will be solved one way or another with v8.2.0 release (ETA: next week)

**Comment by dochang** [comment]
Ok I will check v8.2.0 later. I close this issue at this moment.

Thank you for your help.
